python3-Markdown.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What is it needed for?

Reply:
Seems to be a bug in flatpak-pip-generator. Added it manually now.